src/utils/LocalStorageManager.ts

```diff
@@ -0,0 +1,66 @@
+import { User } from "../types/userTypes";
+
+export class LocalStorageManager {
+  private static readonly USER_ID_KEY = "user_id";
+  private static readonly USER_NAME_KEY = "user_name";
+  private static readonly PROJECT_ID_KEY = "projectId";
+  private static readonly TOKEN_KEY = "token_access";
+
+  static getUserId(): number {
+    return Number(localStorage.getItem(this.USER_ID_KEY));
+  }
+
+  static getUserName(): string {
+    return localStorage.getItem(this.USER_NAME_KEY) || "";
+  }
+
+  static setUser(user: User | undefined): void {
```

Comment:
Type union with `undefined` can be written as `user?: User` -- please check the other occurences with the same pattern as well